zqd/api/connection.go

```diff
@@ -137,6 +137,29 @@ func (c *Connection) Ping(ctx context.Context) (time.Duration, error) {
 	return resp.Time(), nil
 }
 
+// backward compatibility with old versions of zqd
+type VersionCompat struct {
+	VersionResponse
+	Zqd string `json:"boomd"`
+	Zq  string `json:"zq"`
+}
```

Comment:
If brim isn't using the `/version` endpoint can we just get rid of this?

Reply:
Yes, I wasn't sure what used it so out this here.  We can take it out.  I grepped for /version in the brim source and didn't find it.